src/test/java/com/pubnub/api/managers/SubscriptionManagerTest.java

```diff
@@ -67,7 +67,7 @@ public void beforeEach() throws IOException, PubNubException {
 
     @After
     public void afterEach() {
-        pubnub.destroy();
+        pubnub.forceDestroy();
```

Comment:
I found that some tests were flaky without this change and weren't passing for me, as the same pubnub client would be active between tests. This seems to fix it